tools/pirlo-tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+echo "Downloading pirlo-cli tool"
+curl https://pirlo-downloads.s3-us-west-1.amazonaws.com/pirlo-cli/pirlo-cli -o pirlo-cli
+
+chmod a+x pirlo-cli
+
+echo "Initiating a new automated test run."
+
+./pirlo-cli run -k jNmWe3iFhWjvSy_SCjlvLsCWgfw=  android/app/build/outputs/apk/release/app-release.apk
```

Comment:
What does this token mean -- is it a form of auth, and if so what's the scope of it?

One thing we can do if there needs to be a secret involved (because this is a public repo) is IIRC I can put it in Travis config so it gets passed as an environment variable.

Reply:
Yes this is an auth token with a limited scope to be used by CI system. The auth token can only be used for initiating an automated test run and getting its status. We also agree its better idea to add it as an environment variable in your travis config. You can set an environment variable name `PIRLO_API_KEY` and then we don't need to pass it  as cli argument.